### PR TITLE
Task#3/penava

### PR DESCRIPTION
**Pull request by sapjoldaccount**
Early version of sign-in page with Sladic's skeleton incorporated.

### CODE REVIEW

File: src/pages/SportsPageNHL.js [hunk 1]
```diff
@@ -0,0 +1,29 @@
+/** main import(s)                                                      */
+import React, {Component} from 'react';
+
+/** helper import(s)                                                    */
+
+/** component import(s)                                                 */
+
+/** style import(s)                                                     */
+
```
Comment (karthiknayak02) [inline src/pages/SportsPageNHL.js]: remove comments if not needed

File: src/pages/SportsPageNFL.js [hunk 2]
```diff
@@ -0,0 +1,29 @@
+/** main import(s)                                                      */
+import React, {Component} from 'react';
+
+/** helper import(s)                                                    */
+
+/** component import(s)                                                 */
+
+/** style import(s)                                                     */
+
```
Comment (karthiknayak02) [inline src/pages/SportsPageNFL.js]: remove comments if not needed

File: src/pages/SportsPageNBA.js [hunk 3]
```diff
@@ -0,0 +1,29 @@
+/** main import(s)                                                      */
+import React, {Component} from 'react';
+
+/** helper import(s)                                                    */
+
+/** component import(s)                                                 */
+
+/** style import(s)                                                     */
```
Comment (karthiknayak02) [inline src/pages/SportsPageNBA.js]: remove comments if not needed

**Comment by karthiknayak02** [comment]
Add screenshot to PR if possible

File: src/pages/SportsPageNBA.js (hunk 3 above)
Comment (sapjoldaccount) [reply src/pages/SportsPageNBA.js]: done

File: src/_smart-components/Navigation.js [hunk 4]
```diff
@@ -0,0 +1,72 @@
...
+						 </NavItem>
+						 <NavItem className="nav-link-wrappers" >
+							 <Link className="link" to = "/sport/nhl">NHL</Link>
+						 </NavItem>
+						 <NavItem className="nav-link-wrappers" >
+							 <Link className="link" to = "/sport/mlb">MLB</Link>
+						 </NavItem>
+						 <NavItem className="nav-link-wrappers" >
+							 <Link className="link" to = "/sport/mls">MLS</Link>
+						 </NavItem>
```
Comment (karthiknayak02) [inline src/_smart-components/Navigation.js]: Are you using a code formatter? this looks off

File: src/pages/Landing.js [hunk 5]
```diff
@@ -0,0 +1,32 @@
+/** main import(s)                                                      */
+import React, {Component} from 'react';
+
+/** helper import(s)                                                    */
+
+/** component import(s)                                                 */
+
+/** style import(s)                                                     */
+
```
Comment (karthiknayak02) [inline src/pages/Landing.js]: remove these too

File: src/index.js [hunk 6]
```diff
@@ -0,0 +1,31 @@
...
+import './styles/index.css';
+import 'bootstrap/dist/css/bootstrap.min.css';
+
+/*
+  ADD LATER: REDUX
+  <Provider store={store}>
+
+  </Provider>
+*/
+
```
Comment (karthiknayak02) [inline src/index.js]: Make this a issue instead of having it in comments

File: src/pages/SignupLogin.js [hunk 7]
```diff
@@ -0,0 +1,88 @@
+/** main import(s)                                                      */
+import React, {Component} from 'react';
+
+/** helper import(s)                                                    */
+
+/** component import(s)                                                 */
+
```
Comment (karthiknayak02) [inline src/pages/SignupLogin.js]: remove these

File: src/pages/SportsPageMLB.js [hunk 8]
```diff
@@ -0,0 +1,29 @@
+/** main import(s)                                                      */
+import React, {Component} from 'react';
+
+/** helper import(s)                                                    */
+
+/** component import(s)                                                 */
+
+/** style import(s)                                                     */
+
```
Comment (karthiknayak02) [inline src/pages/SportsPageMLB.js]: and this

File: src/pages/SportsPageMLS.js [hunk 9]
```diff
@@ -0,0 +1,29 @@
+/** main import(s)                                                      */
+import React, {Component} from 'react';
+
+/** helper import(s)                                                    */
+
+/** component import(s)                                                 */
+
+/** style import(s)                                                     */
+
```
Comment (karthiknayak02) [inline src/pages/SportsPageMLS.js]: and this

**Comment by sapjoldaccount** [comment]
Just want to get this merged so we have a good first commit in master.

**Comment by Nickadiemus** [comment]
Good Merge to Master

**Review by Nickadiemus — APPROVED** [review]
Good Merge

**Pull request merged**